tests/python/gaia-ui-tests/gaiatest/tests/functional/contacts/test_import_edit_export_contact.py

```diff
@@ -11,8 +11,10 @@ def setUp(self):
         GaiaTestCase.setUp(self)
         curr_time = repr(time.time()).replace('.', '')
 
-        self.contact = MockContact(givenName='Name%s'%curr_time[12:], name='Name%s'%curr_time[12:], familyName='')
+        for contact in self.data_layer.sim_contacts:
```

Comment:
Same as what jlorenzo says, we do wipe the content of the device at the beginning of each test, but wiping sim sounds a bit risky to me.  AFAIK, we don't wipe the SD card contents either, do we now?